docker/cas-airflow/Dockerfile

```diff
@@ -4,14 +4,28 @@
 # TODO: build a real image properly, instead of pushing this nasty hack directly to the registry
 # see https://registry-console-default.pathfinder.gov.bc.ca/registry#/images/wksv3k-tools/cas-airflow
 
-FROM puckel/docker-airflow:1.10.4
-LABEL io.openshift.s2i.assemble-user="1000"
+FROM puckel/docker-airflow:1.10.9
+# Never prompt the user for choices on installation/configuration of packages
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM linux
+LABEL io.openshift.s2i.assemble-user="1000880000"
 USER root
+RUN set -ex \
+    && apt-get update -yqq \
+    && apt-get upgrade -yqq \
+    && apt-get install python-lxml -yqq \
+    && pip install lxml \
+    && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql,ssh,gcp,kubernetes]==1.10.10
 COPY root/entrypoint.sh /entrypoint.sh
-COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
-RUN test "$(id airflow)" = "uid=1000(airflow) gid=1000(airflow) groups=1000(airflow)" && \
-    chmod g=u /etc/passwd && \
+RUN userdel airflow && \
+    groupadd -g 1000880000 airflow && \
+    useradd -l -u 1000880000 -g 1000880000 -d /usr/local/airflow airflow && \
```

Comment:
This is binding the `/usr/local/airflow` home folder to the new airflow user but leaving the permissions on that folder as owned by the now-deleted original airflow user. I couldn't figure out a way around this as docker refuses to change the ownership of a volume after instantiation and we're doing a `FROM` instruction where the volume has already been defined. Another reason to build the image properly here...